### DISCUSSION TITLE
Can I have Docker support?

### DISCUSSION THREAD

**Discussion by armoreno** [Joshua-Riek/ubuntu-rockchip#1145] [General]
I have an app running inside a Docker container, and now I need to run it on a hardware platform based on RK3588. It would be great if it supported Docker containers. 
Could someone confirm it? Thank you!

**Comment by L2jLiga** [comment]
Hi, I'm using Orange Pi 5 with Ubuntu 24.04 as home lab kubernetes node with containerd.io from docker ppa

**Reply by nrpetonr** [reply]
sure it ddoes

**Reply by armoreno** [reply]
Thank you for your confirmation!
